src/main/webapp/resources/js/components/samples/components/SampleAnalyses.tsx

```diff
@@ -1,14 +1,17 @@
 import React from "react";
-import { Col, Input, Row, Table, Typography } from "antd";
-import { useDispatch, useSelector } from "react-redux";
+import { Col, Input, Row, Table } from "antd";
+import { useAppDispatch, useAppSelector } from "../../../hooks/useState";
 import { fetchSampleAnalyses } from "../../../apis/samples/samples";
 import { setSampleAnalyses } from "../sampleAnalysesSlice";
 import { SampleAnalysesState } from "./SampleAnalysesState";
 import { setBaseUrl } from "../../../utilities/url-utilities";
 import { formatInternationalizedDateTime } from "../../../utilities/date-utilities";
+import {
+  AnalysisState,
+  SampleAnalyses as SampleAnalysesItem,
+} from "../../../apis/samples/samples";
```

Comment:
Clean up your imports, you have multiple from this one.

Reply:
Cleaned up in 1398b80e9ff839bad02c6c0dfd63fabd2e3bf8a5